index.md

```diff
@@ -3,6 +3,14 @@ layout: default
 title:  JuliaOpt&#58; Optimization packages for the Julia language
 ---
 
+# JuMP-dev IV
```

Comment:
I think "4th annual JuMP-dev" is a bit more standard. "IV" makes me think of rocky or star trek.